src/simulation_strategies/GeneralizedCliffordsSimulationStrategy.py

```diff
@@ -0,0 +1,164 @@
+__author__ = 'Tomasz Rybotycki'
```

Comment:
Nitpick: `typing` is largely missing in methods in this file.

Reply:
Fixed.